design-system/materials/internals/definition.yaml

```diff
@@ -329,6 +335,8 @@ decisionGroupsByTheme:
         border-radius-for-tag:
           description: ''
           choice: border-radius-2
+        border-radius-for-card:
```

Comment:
This token is inside the `default` theme group so the value here should be `border-radius-6` (the current value).
Also, you would need to add this token in the test group with this value (`border-radius-4`).